test/test.js

```diff
@@ -41,226 +35,215 @@ test('error if no url is specified', t => {
 
 	new Pageres()
 		.src('', [])
-		.run(err => {
+		.run()
+		.catch(err => {
 			t.ok(err);
 			t.is(err.message, 'URL required');
 		});
 });
 
 test('generate screenshots', t => {
-	t.plan(5);
+	t.plan(4);
 
-	const pageres = new Pageres()
+	new Pageres()
 		.src('yeoman.io', ['480x320', '1024x768', 'iphone 5s'])
-		.src('todomvc.com', ['1280x1024', '1920x1080']);
-
-	pageres.run((err, streams) => {
-		t.ifError(err);
-		t.is(streams.length, 5);
-		t.is(streams[0].filename, 'todomvc.com-1280x1024.png');
-		t.is(streams[4].filename, 'yeoman.io-320x568.png');
-		streams[0].once('data', data => t.true(data.length > 1000));
-	});
+		.src('todomvc.com', ['1280x1024', '1920x1080'])
+		.run()
+		.then(streams => {
+			t.is(streams.length, 5);
+			t.is(streams[0].filename, 'todomvc.com-1280x1024.png');
+			t.is(streams[4].filename, 'yeoman.io-320x568.png');
+			streams[0].once('data', data => t.true(data.length > 1000));
+		});
 });
 
 test('remove special characters from the URL to create a valid filename', t => {
-	t.plan(3);
-
-	const pageres = new Pageres()
-		.src('http://www.microsoft.com/?query=pageres*|<>:"\\', ['1024x768']);
+	t.plan(2);
 
-	pageres.run((err, streams) => {
-		t.ifError(err);
-		t.is(streams.length, 1);
-		t.is(streams[0].filename, 'microsoft.com!query=pageres-1024x768.png');
-	});
+	new Pageres()
+		.src('http://www.microsoft.com/?query=pageres*|<>:"\\', ['1024x768'])
+		.run()
+		.then(streams => {
```

Comment:
Happy to have this done in a follow-up PR if you prefer, but would be super nice to use async/await in the tests.

https://github.com/sindresorhus/temp-write/blob/985ea544148bdcd15df4d5e960ddb6c0772c4194/test.js#L7